CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.1.0] - 2024-11-13
+
+### Changed
+
+- v16: ExtrinsicMetadata extensions [#86](https://github.com/paritytech/frame-metadata/pull/86)
```

Comment:
isn't this a breaking change i.e. some public fields of the ExtrinsicMetadata is changed?

Reply:
Yeah it's tough. The "unstable" means we are allowed to break it without caring, but that sort of approach does also mean any crate depending on the unstable feature would start breaking until updated. (eg polkadot-sdk and merkleized-metadata)

Reply:
Hmm yep, its a tough call, in the past we had breaking change for unstable versions (ie v15.1 included v15unstable). But since we cut a v17.0 for v16 unstable, I think its fine to do a v18 as well (just to keep consistency with our latest release schedules)